packages/logger/tests/unit/middleware/middy.test.ts

```diff
@@ -9,15 +9,15 @@ import { injectLambdaContext } from '../../../src/middleware/middy';
 import { Logger } from './../../../src';
 import middy from '@middy/core';
 import { PowertoolLogFormatter } from '../../../src/formatter';
+import { Console } from 'console';
 
 describe('Middy middleware', () => {
 
   const ENVIRONMENT_VARIABLES = process.env;
 
   beforeEach(() => {
     jest.resetModules();
-    // eslint-disable-next-line @typescript-eslint/no-empty-function
-    jest.spyOn(console, 'log').mockImplementation(() => {});
+    jest.spyOn(process.stdout, 'write').mockImplementation(() => null as unknown as boolean);
```

Comment:
Why double cast? Is this intentional?

Reply:
It's only to please the TypeScript compiler.

If doing `null as boolean` it would give me this error:
```sh
error TS2352: Conversion of type 'null' to type 'boolean' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.

jest.spyOn(process.stdout, 'write').mockImplementation(() => null as boolean);
```

Reply:
Got it. Thanks.